### PR TITLE
added alt ratings for gracenote

### PR DESCRIPTION
**Pull request by gaurav-amagi**
Added Gracenote ratings to alt label tag

Json getting genreated using the amgrss tool

image

### CODE REVIEW

File: skos/ebu_ParentalGuidanceCodeCS_amgext.rdf [hunk 1]
```diff
@@ -968,6 +970,7 @@
       rdf:about="http://www.ebu.ch/metadata/ontologies/skos/ebu_ParentalGuidanceCodeCS#_7">
       <rdf:type rdf:resource="http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#TargetAudience"/>
       <skos:prefLabel xml:lang="en">BR-DJCTQ</skos:prefLabel>
+      <skos:altLabel xml:lang="pt-BR">Departamento de JustiÃ§a, ClassificaÃ§Ã£o, TÃ­tulos e QualificaÃ§Ã£o</skos:altLabel>
```
Comment (shashi-banger) [inline skos/ebu_ParentalGuidanceCodeCS_amgext.rdf:973]: @gaurav-amagi some unicode character issue here "Departamento de Justiça, Classificação, Títulos e Qualificação "
https://www.jusbrasil.com.br/artigos/busca?q=DEPARTAMENTO+DE+JUSTI%C3%87A%2C+CLASSIFICA%C3%87%C3%83O%2C+T%C3%8DTULOS+E+QUALIFICA%C3%87%C3%83O

File: skos/ebu_ParentalGuidanceCodeCS_amgext.rdf [hunk 2]
```diff
@@ -1753,6 +1759,7 @@
       <rdf:type rdf:resource="http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#TargetAudience"/>
       <skos:prefLabel xml:lang="es">CL-ANATEL</skos:prefLabel>
       <skos:prefLabel xml:lang="en">CL-ANATEL</skos:prefLabel>
+      <skos:altLabel xml:lang="es">Consejo Nacional de TelevisiÃ³n</skos:altLabel>
```
Comment (shashi-banger) [inline skos/ebu_ParentalGuidanceCodeCS_amgext.rdf:1762]: Again some issue with unicode
Consejo Nacional de Televisión

File: skos/ebu_ParentalGuidanceCodeCS_amgext.rdf [hunk 3]
```diff
@@ -5697,6 +5721,7 @@
       rdf:about="http://www.ebu.ch/metadata/ontologies/skos/ebu_ParentalGuidanceCodeCS#_44">
       <rdf:type rdf:resource="http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#TargetAudience"/>
       <skos:prefLabel xml:lang="en">SE-SB</skos:prefLabel>
+      <skos:altLabel xml:lang="sv">Statens medierÃ¥d</skos:altLabel>
```
Comment (shashi-banger) [inline skos/ebu_ParentalGuidanceCodeCS_amgext.rdf:5724]: @gaurav-amagi Statens medieråd
Reference: https://rating-system.fandom.com/wiki/Swedish_Media_Council

**Review by shashi-banger — APPROVED** [review]

File: skos/ebu_ParentalGuidanceCodeCS_amgext.rdf (hunk 1 above)
Comment (gaurav-amagi) [reply skos/ebu_ParentalGuidanceCodeCS_amgext.rdf:973]: Yes @shashi-banger we have asked Gracenote for a better list to avoid this problem

**Comment by shashi-banger** [comment]
Closing this PR since it is being addressed in #81

**Comment by shashi-banger** [comment]
Closing this PR since it is being addressed in https://github.com/amagimedia/metadata/pull/81

**Pull request closed without merging**